src/api/libcellml/component.h

```diff
@@ -1,11 +1,24 @@
+/*
+Copyright 2015 University of Auckland
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.Some license of other
+*/
+
 #ifndef LIBCELLML_LIBCELLML_COMPONENT_H
 #define LIBCELLML_LIBCELLML_COMPONENT_H
 
-#include <string>
-#include <memory>
-
 #include "libcellml/libcellml_export.h"
```

Comment:
Besides listing includes in alphabetical order (and the same throughout the code), I was wondering whether it wouldn't be better to have ```export.h``` (and ```config.h```) rather than ```libcellml_export.h``` (and ```libcellml_config.h```)? I just don't see the point of having ```libcellml_``` in front, not to mention that using ```_``` kind of breaks our current file name convention, no?